### PR TITLE
DB Refactoring

### PR DESCRIPTION
**Pull request by BenoitSafari**
Reworked DB files and structure, prepared seed files for V2 migration. Also made a few fixes on API/APP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ PGADMIN_DEFAULT_PASSWORD=
 
 ### .env.app:
 ```
+NEXT_PUBLIC_PASS_REGEXP=
 NEXT_PUBLIC_TMDB_KEY=
```
Comment (BenoitSafari) [inline README.md:29]: Don't forget to update your `env` file

File: app/components/Forms/InscriptionForm.tsx [hunk 2]
```diff
@@ -89,15 +89,15 @@ export default function InscriptionForm() {
...
+          pattern={process.env.NEXT_PUBLIC_PASS_REGEXP}
           errorMessage="La saisie ne réponds pas aux exigences de sécurité."
           ref={PWRef}
         />
         <TextInputRef
           type={!isPWVisible ? 'password' : undefined}
           id="password"
           placeholder="Confirmer votre mot de passe..."
-          pattern="^(?=.*[A-Za-z])(?=.*\d)[!#$&%*+=?|\-A-Za-z\d]{8,}$"
+          pattern={process.env.NEXT_PUBLIC_PASS_REGEXP}
```
Comment (BenoitSafari) [inline app/components/Forms/InscriptionForm.tsx:100]: `pattern` does not share the same syntax as JS string regexp for some reason. You can either convert the regex to the correct format or create a function that check the field onChange. That's why we don't have the same behavior on APP/API.

Comment (BenoitSafari) [inline app/components/Forms/InscriptionForm.tsx:100]: `pattern` does not share the same syntax as JS string regexp for some reason. You can either convert the regex to the correct format or create a function that check the field onChange. That's why we don't have the same behavior on APP/API.

File: data/V02-SEED_USER.sql [hunk 3]
```diff
@@ -0,0 +1,64 @@
+-- TEST USERS SEEDS
+-- PASSWORD: 'Cinevoraces2022!'
+-- role: 0 = user, 1 = moderator, 2 = admin
```
Comment (BenoitSafari) [inline data/V02-SEED_USER.sql:3]: Every user share the same password in dev environment.

**Review by Julien-Goletto — APPROVED** [review]

**Pull request merged**